content/en/docs/devops-user-guide/how-to-integrate/harbor.md

```diff
@@ -93,15 +124,15 @@ helm install harbor-release harbor/harbor --set expose.type=nodePort,externalURL
        // docker image name
        APP_NAME = 'docker-example'
        // ‘yuswift’ is the credential id you created on ks console
```

Comment:
we should replace yuswift with robot-test

Reply:
Good catch. Thanks for your comment.